From 1d3cee822a66d1201cdb51df0df4277a4cd7be47 Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Fri, 6 Jun 2025 13:04:15 +0530
Subject: [PATCH 6/6] sbi: init: Modify CSR registers

Changed CSR 0x7C3 and 0x7C4 values to improve DDR preformance
  0x7C3 --> 0x104095c1be241
  0x7C4 --> 0x93FF

Upstream-Status: Pending

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Signed-off-by: Pritesh Patel <pritesh.patel@einfochips.com>
---
 lib/sbi/sbi_init.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/sbi/sbi_init.c b/lib/sbi/sbi_init.c
index 2952601..402325b 100644
--- a/lib/sbi/sbi_init.c
+++ b/lib/sbi/sbi_init.c
@@ -576,11 +576,11 @@ void __noreturn sbi_init(struct sbi_scratch *scratch)
 	/* Force noisy evict to send release message from any valid coherence permission state */
 	csr_write(0x7C2, 0x80);
 
-	/* Set CSR 0x7C3 to 0x5c1be649 */
-	csr_write(0x7C3, 0x5c1be649);
+	/* Set CSR 0x7C3 to 0x104095c1be241 */
+	csr_write(0x7C3, 0x104095c1be241);
 
-	/* Set CSR 0x7C4 to 0x929F */
-	csr_write(0x7C4, 0x929f);
+	/* Set CSR 0x7C4 to 0x93FF */
+	csr_write(0x7C4, 0x93ff);
 
 #endif
 
-- 
2.25.1

